### ISSUE TITLE
Youtube - Unable to extract uploader id

### ISSUE THREAD

**Issue by zxmon21** [ytdl-org/youtube-dl#31717]
Hi,
upon 
youtube-dl https://www.youtube.com/watch?v=8XWAktvFZGI --verbose

i get this output:
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['https://www.youtube.com/watch?v=8XWAktvFZGI', '--verbose']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Python version 3.10.6 (CPython) - Linux-5.19.0-32-generic-x86_64-with-glibc2.35
[debug] exe versions: ffmpeg 4.4.2, ffprobe 4.4.2, rtmpdump 2.4
[debug] Proxy map: {}
[youtube] 8XWAktvFZGI: Downloading webpage
ERROR: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/youtube_dl/YoutubeDL.py", line 815, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/youtube_dl/YoutubeDL.py", line 836, in __extract_info
    ie_result = ie.extract(url)
  File "/usr/lib/python3/dist-packages/youtube_dl/extractor/common.py", line 534, in extract
    ie_result = self._real_extract(url)
  File "/usr/lib/python3/dist-packages/youtube_dl/extractor/youtube.py", line 1794, in _real_extract
    'uploader_id': self._search_regex(r'/(?:channel|user)/([^/?&#]+)', owner_profile_url, 'uploader id') if owner_profile_url else None,
  File "/usr/lib/python3/dist-packages/youtube_dl/extractor/common.py", line 1012, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dl.utils.RegexNotFoundError: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

I've tried this on a Ubuntu (v2021.12.17) and Windows (2021.12.17) computer. Both are frequently updated. I've tried with 4 random videos, with the same result

**Comment by linux-rox** [comment]
This is a known issue: https://github.com/ytdl-org/youtube-dl/issues/31530

See links for workarounds.  What worked for me was pip install --upgrade git+https://github.com/ytdl-org/youtube-dl.git@master.  You may have to install pip and/or git.  You might also have to adjust your $PATH variable, if (as in my case) the upgrade is downloaded to $HOME/.local/bin.  You do that by editing .bashrc or .profile.

**Comment by FireTonnor** [comment]
I was not able to get it to work in any way so what I would suggest if you want to do it is switch to [yt-dlp](https://github.com/yt-dlp/yt-dlp) when you install it from the release you will have to make a folder in your files and rename the yt-dlp file to youtube-dl if you are using mpv and add the path to your system environment path also this only works for windows

**Comment by FireTonnor** [comment]
@zxmon21 @linux-rox

**Comment by zxmon21** [comment]
Duplicate

**Comment by Vangelis66** [comment]
@FireTonnor wrote:

> I was not able to get it to work **in any way**... also this **only works for windows**

... Pray tell why 
```
https://github.com/ytdl-patched/youtube-dl/releases/latest/download/youtube-dl.exe
```
**didn't work for you under Windows**? 😞 One doesn't necessarily have to migrate to `yt-dlp` just for the sake of this specific `ERROR->WARNING->now fixed` ... 😡 
Additionally, instead of high-jacking the 150th duplicate, you should have really started by visiting the PINNED issue #30839 (inside it, a required search for dupes would've landed you to #31530; inside which solutions for Windows, including the one I cited above, are to be found) ...
